components/dataViews/AccountView/BalancesList.tsx

```diff
@@ -0,0 +1,46 @@
+import { Card, CardContent, CardHeader, CardTitle } from "@/components/ui/card";
+import { Coin } from "@cosmjs/amino";
+import { StargateClient } from "@cosmjs/stargate";
+import { Dispatch, SetStateAction, useEffect, useState } from "react";
+import { useChains } from "../../../context/ChainsContext";
+import BalancePill from "./BalancePill";
+
+interface BalancesListProps {
+  readonly walletAddress: string;
+  readonly setError: Dispatch<SetStateAction<string>>;
+}
+
+export default function BalancesList({ walletAddress, setError }: BalancesListProps) {
+  const { chain } = useChains();
+  const [balances, setBalances] = useState<readonly Coin[]>([]);
+
+  useEffect(() => {
+    (async function () {
+      if (!walletAddress) {
+        return;
+      }
+
+      try {
+        const client = await StargateClient.connect(chain.nodeAddress);
+        const newBalances = await client.getAllBalances(walletAddress);
+        setBalances(newBalances);
+      } catch (e: unknown) {
+        setError(e instanceof Error ? e.message : "Failed to get balances");
+        console.error("Get balances error:", e);
+      }
+    })();
+  }, [chain.nodeAddress, setError, walletAddress]);
+
+  return balances.length ? (
```

Comment:
How do I get this balance tab? I cannot really verify it on the preview. I assume it requires me to connect kepler, but connecting to kepler (at least on preview) seems not to work.